README.md

```diff
@@ -321,9 +321,9 @@ pip install -r requirements.txt
 
 Start mining! `python run.py` 🥳
 
-### <s>pip</s> deprecated
-<s>Install the package via pip, you will find a stable version - maybe a different version from the master branch.
-- `pip install Twitch-Channel-Points-Miner-v2`
+### pip
+Install the package via pip.
```

Comment:
Error using pip install (Windows 64bit, anaconda/spyder):

pip install "git+https://github.com/rdavydov/Twitch-Channel-Points-Miner-v2.git"
  ERROR: Error [WinError 2] The system cannot find the file specified while executing command git versionCollecting git+https://github.com/rdavydov/Twitch-Channel-Points-Miner-v2.gitNote: you may need to restart the kernel to use updated packages.


  Cloning https://github.com/rdavydov/Twitch-Channel-Points-Miner-v2.git to c:\users\patri\appdata\local\temp\pip-req-build-xrn0f3z_ERROR: Cannot find command 'git' - do you have 'git' installed and in your PATH?